### PR TITLE
Extend vsc tutorials description

### PR DESCRIPTION
**Pull request by moravveji**
This PR follows #365 . I noticed that the trainings offered by other HPC sites are thoroughly documented here https://docs.vscentrum.be/interesting_links.html#training-programs-in-other-centres. However, there was no single mention of our own official VSC training page on the docs. So, this PR:

- follows that of #365 
- tries to promote VSC trainings (but a single sentence may not be enough; so, feel free to propose any improvements here)
- describes that the KUL/UH trainings are spread over various github repositories (in contrast to those of other sites which are nicely organized under a single website)
- proposed changes are only for the KUL/UH site

### CODE REVIEW

File: source/vsc_tutorials.rst [hunk 1]
```diff
@@ -5,14 +5,24 @@ VSC tutorials
...
+user portal, the latter being the reference manual. Below you can find links 
+to those training materials.
 
 KULeuven/UHasselt
 -----------------
 
+The training material (slides and exercises) for the a variety of HPC-related trainings from
+the KULeuven/UHasselt site are organized via the 
+`official Github repository <https://github.com/hpcleuven>`__. 
+The beginners might consider following:
```
Comment (lexming) [inline source/vsc_tutorials.rst:20]:
```suggestion

New users without prior experience with Linux or the HPC might consider the following introductory tutorials: 
```

File: source/vsc_tutorials.rst [hunk 2]
```diff
@@ -5,14 +5,24 @@ VSC tutorials
...
+
 * `HPC Introduction <https://hpcleuven.github.io/HPC-intro/>`__
+* `Linux for HPC <https://hpcleuven.github.io/Linux-for-HPC/>`__
+* `Linux Scripting <https://hpcleuven.github.io/Linux-scripting/>`__
+* `Linux Tools <https://hpcleuven.github.io/Linux-tools/>`
```
Comment (lexming) [inline source/vsc_tutorials.rst:25]:
```suggestion
* `Linux Tools <https://hpcleuven.github.io/Linux-tools/>__`
```

**Review by lexming — COMMENTED** [review]
Good idea to add a link to https://www.vscentrum.be/vsctraining. I have a couple minor suggestions:

File: source/vsc_tutorials.rst (hunk 1 above)
Comment (moravveji) [reply source/vsc_tutorials.rst]: Thanks @lexming . This is improved in `08efa84`

File: source/vsc_tutorials.rst (hunk 2 above)
Comment (moravveji) [reply source/vsc_tutorials.rst]: fixed in `2db68ce`

**Review by lexming — APPROVED** [review]
LGTM

File: source/vsc_tutorials.rst [hunk 3]
```diff
@@ -5,14 +5,25 @@ VSC tutorials
...
+VSC sites carry out regular trainings about Linux and the VSC HPC systems. 
+You may always find the updated list of `VSC trainings <https://www.vscentrum.be/vsctraining>`__ 
+via the VSC website. It is complementary to the information found in this 
+user portal, the latter being the reference manual. Below you can find links 
+to those training materials.
 
 KULeuven/UHasselt
 -----------------
 
+The training material (slides and exercises) for the a variety of HPC-related trainings from
```
Comment (stevenvdb) [inline source/vsc_tutorials.rst:17]: for the a -> for a

**Review by stevenvdb — COMMENTED** [review]
Apart from a small typo, this is ok for me.

File: source/vsc_tutorials.rst (hunk 3 above)
Comment (moravveji) [reply source/vsc_tutorials.rst]: Thx. Fixed in `256dbb`

**Review by stevenvdb — APPROVED** [review]

**Pull request merged**